pyproject.toml

```diff
@@ -19,10 +19,12 @@ authors = [
 version = "0.0.1"
 
 dependencies = [
-  "xarray",
-  "netcdf4",
-  "numpy",
-  "jupyter"
+  "xarray>=2025.4.0",
```

Comment:
The prior versions of these packages are not compatible? I would leave the `pyproject.toml` as loose as possible and include a ``requirement.txt` if we want to provide a more exact environment specification. Otherwise users with older Python versions may have issues installing the package.